.travis.yml

```diff
@@ -67,13 +65,16 @@ jobs:
             make compile;
             ./tests/e2e_tests build/infection.phar;
           fi
-        - if [[ "$DRIVER" == "none" ]]; then xdebug-disable; fi
+        - if [[ "$DRIVER" == "none" ]]; then xdebug_disable; fi
         - |
-          INFECTION_PR_FLAGS=$(get-infection-pr-flags);
-          echo "INFECTION_PR_FLAGS=$INFECTION_PR_FLAGS";
+          INFECTION_PR_FLAGS=$(get_infection_pr_flags);
+
+          echo "Flags used: ${INFECTION_FLAGS} ${INFECTION_PR_FLAGS};";
 
           if [[ "$DRIVER" != "phpdbg" ]]; then
             bin/infection $INFECTION_FLAGS $INFECTION_PR_FLAGS;
+          elif [[ "$DRIVER" == "none" ]]; then
+            exit 0
```

Comment:
I don't think there is any reason to run infection without phpdbg for the driver "none": it already run all the tests and there is a dedicated build for the phpdbg driver

Reply:
Note to myself: should be place _above_ the previous condition, otherwise will never be executed